function/function.go

```diff
@@ -222,3 +222,26 @@ func (f *Function) callEmulator(payload []byte) ([]byte, error) {
 	defer resp.Body.Close()
 	return ioutil.ReadAll(resp.Body)
 }
+
+func DetermineErrorMessage(err error) string {
```

Comment:
doesn't have to be public function

Reply:
How do you mean? It is being used in a different package (`router`). So, doesnt it have to be?